sippy-ng/src/tests/TestTable.js

```diff
@@ -387,6 +456,16 @@ function TestTable(props) {
     </div>
   )
 
+  const getVariantStyle = (variant) => {
+    // Special treatment for JobTier to help users better understand if their test is feeding component readiness or not
+    if (isComponentReadinessIncludedJobTier(variant)) {
+      return { color: 'green' }
+    } else if (variant.startsWith('JobTier:')) {
+      return { color: 'darkred' }
+    }
```

Comment:
Nit, you can use theme colors instead: `theme.palette.error.dark` and `theme.palette.success.dark` instead.  But this version is also fine, it's just more consistent the other way